broker/config/settings.yml

```diff
@@ -402,6 +402,17 @@ defaults: &defaults
     url: https://api.bosh-lite.com
     username: admin
     password: admin
+  
+  #############################
+  # CREDHUB SETTINGS #
+  #############################
+  cred_provider:
+    credhub_url: https://10.0.3.11:8443/api/
+    credhub_uaa_url: https://10.0.3.11:8443
+    credhub_client_id:  credhub-admin
+    credhub_client_secret: 
```

Comment:
Does this mean there will be a single uaa_client_id/secret for all the services on-boarded? What if each service has its own dedicated clientid/secret in-line with what we have in place for uaa username/password for token retrieval?

Reply:
Same question.

Reply:
No. Each service has its own credential. This is master credentials that SF uses and it can fetch any creds. Individual services can just fetch their creds. Please review sf-produc-component PR to know about individual service users/creds creation.